css/quotes.css

```diff
@@ -1,185 +1,271 @@
-* in this case the class "form__label".
-* Note: the order is established in the html.
-*/
-.form__input:focus + .form__label {
-  margin-top: -86px;
-  color: #0086c3;
+ * In this code, i assign properties to the element that is after to the class "form__input",
+ * in this case the class "form__label".
+ * Note: the order is established in the html.
+ */
+.form-field__input:focus + .form-field__label {
+  margin-top: -65px;
+  color: #29b6f6;
 }
 
-.form__input {
+.form-field__input {
   border: 0;
   border-bottom: 1px solid #9e9e9e;
-  font-size: 12px;
-  font-weight: lighter;
-  margin-right: 5%;
-  margin-left: 5%;
+  font-size: 14px;
   padding: 6px;
   background: rgba(0, 0, 0, 0);
-  margin-bottom: 40px;
+  margin-block-start: 20px;
+  margin-block-end: 20px;
   display: block;
-  width: 90%;
+  width: 100%;
 }
 
-.form__label {
-  font-size: 11px;
-  font-weight: lighter;
-  padding-left: 15px;
+.form-field__label {
+  font-size: 12px;
   position: absolute;
-  margin-top: -65px;
+  margin-top: -45px;
   z-index: -20;
   display: block;
-  width: 90%;
+  width: inherit;
   color: rgba(0, 0, 0, 0.5);
   transition: all 0.2s;
-  cursor: pointer;
 }
 
 .fix-label {
-  margin-top: -86px;
+  margin-top: -65px;
 }
 
-.div__options {
+.form__options {
   margin-top: 40px;
 }
 
-.div__quote {
+.form {
   margin-bottom: 50px;
 }
 
-.checkbox__container input {
+.form-options__label input {
   display: none;
 }
 
-.checkbox__container {
+.form-options__label {
   display: block;
-  padding-left: 26px;
+  padding-left: 35px;
   position: relative;
-  cursor: pointer;
-  font-size: 12px;
   user-select: none;
-  margin-bottom: 9px;
+  margin-block-start: 5px;
+  margin-block-end: 5px;
+  font-size: 14px;
+  font-weight: 300;
+  height: 25px;
 }
 
-.checkbox__container .checkmark {
+.form-options__label .form-options__checkmark {
   display: inline-block;
-  width: 17px;
-  height: 17px;
-  background: #eee;
+  width: 25px;
+  height: 25px;
   position: absolute;
   left: 0;
   top: 0;
-  border-radius: 2px;
+  border: solid 2px #979797;
+  border-radius: 1px;
 }
 
 /*
-* In this code i assign the background color to class
-* checkbox__container inputs that are selected.
-*/
-.checkbox__container input:checked + .checkmark {
-  background-color: #0086c3;
+ * In this code i assign the background color to class
+ * checkbox__container inputs that are selected.
+ */
+.form-options__label input:checked + .form-options__checkmark {
+  background-color: #1565c0;
+  border: solid 2px #1565c0;
 }
 
 /*
-* In this code insert the figure "✓" after that a checkbox is selected; the figure is created
-* assigning attributes and properties to the content.
-*/
-.checkbox__container input:checked + .checkmark:after {
+ * In this code insert the figure "✓" after that a checkbox is selected; the figure is created
+ * assigning attributes and properties to the content.
+ */
+.form-options__label input:checked + .form-options__checkmark:after {
```

Comment:
please don't use tags like `input` for giving class...

Reply:
como esto puede ser algo confuso...

Por favor no usen tags para asignar apariencia, en este caso `input`, recurran a las clases unicamente.

si esto aun es confuso escribanme.